libmscore/paste.cpp

```diff
@@ -482,7 +483,7 @@ void Score::pasteChordRest(ChordRest* cr, int tick, const Interval& srcTranspose
       // we have already disallowed a tuplet from crossing the barline, so there is no problem here
       // but due to rounding, it might appear from actualTicks() that the last note is too long by a couple of ticks
 
-      if (!isGrace && !cr->tuplet() && (tick + cr->actualTicks() > measureEnd || convertMeasureRest)) {
+      if (!isGrace && !cr->tuplet() && (tick + cr->actualTicks() > measureEnd || cr->durationTypeTicks() != cr->actualTicks() || convertMeasureRest)) {
```

Comment:
It is good to have additional parenthesizes not to remember the operations sequence. If something like || vs && vs == is clear, but + in this sequence makes things worse. Could you please add the parenthesizes?